config/multiBandDriver.py

```diff
@@ -0,0 +1,8 @@
+import os.path
+
+from lsst.utils import getPackageDir
+
+for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd"):
+    path = os.path.join(getPackageDir("obs_decam"), "config", sub + ".py")
+    if os.path.exists(path):
+        getattr(config, sub).load(path)
```

Comment:
This doesn't need to be addressed here, but I noticed that this same code is in the obs_subaru config override.  Does it make sense to push this into the task config?  I'm not sure how to do it, but it always worries me to see this much duplicated code.